assets/common/src/deploy.py

```diff
@@ -262,10 +263,15 @@ def main():
     # get registered model id
     if args.registration_details:
         model_info = {}
-        with open(args.registration_details) as f:
-            model_info = json.load(f)
-        model_id = model_info["id"]
-        model_name = model_info["name"]
+
+        registration_details_file = args.registration_details/"model_registration_details.json"
+        try:
```

Comment:
Could you verify for following cases:
1. registration_details_folder  exists, but `model_registration_details.json` doesn't and `model_id` is passed
2. registration_details_folder  exists, but `model_registration_details.json` doesn't and `model_id` is not passed
3. registration_details_folder  and `model_registration_details.json` exists

Reply:
Added.